package.json

```diff
@@ -6,8 +6,8 @@
 		"dev": "npm run protos && vite",
 		"build": "npm run protos && tsc && vite build",
 		"preview": "vite preview",
-		"lint": "eslint . --ext .ts,.tsx",
-		"prettier:check": "prettier -c .",
+		"lint": "eslint . --ext .ts,.tsx && prettier -c .",
+		"lint:fix": "eslint . --ext .ts,.tsx --fix && prettier -w .",
```

Comment:
Might as well add `--ext proto` here while we're at it?

Reply:
So I tried but it fails out of the box. I think this needs a bit more investigation.

So far I found there is this thing https://github.com/yoheimuta/protolint